src/components/pet/pet.component.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, Fragment } from 'react';
+
+import { ObservationsComponent } from './components/observations/observations.component';
+import { AdditionalComponent } from './components/additional/additional.component';
+import { CharacteristicsComponent } from './components/characteristics/characteristics.component';
+import { Logo } from './components/logo/logo.component';
+import './pet.css';
+
+interface PetProps {
+  match: {
+    params: {
+      id: number;
+    };
+  };
+}
+export const Pet = (props: PetProps): React.ReactElement => {
+  const petId: number = props.match.params.id;
+  const [imgUrl, setImgUrl] = React.useState<string>('');
+  const [breed, setBreed] = React.useState<string>('');
+  const [characteristics, setCharacteristics] = React.useState();
+  const [observations, setObservations] = React.useState();
+  const [additionalInfo, setAdditionalInfo] = React.useState();
+  const [loading, setLoading] = React.useState<boolean>(true);
+
+  async function getPet(id: number) {
+    const dogProfile = await fetch(`http://localhost:5000/pets/${id}`);
+    if (dogProfile.ok) {
+      return await dogProfile.json();
+    } else {
+      return Promise.reject();
+    }
+  }
```

Comment:
I thing it would be better to use Saga and write data to global State using Redux

Reply:
Not sure if it is necessary to put this data in the global state, as it is not expected to be reused somewhere else in the application. I`ll seek Roma for guidance and come back to this. Thanks for the note